src/ceph.in

```diff
@@ -342,7 +342,9 @@ def format_help(cmddict, partial=None):
 
         if not cmd['help']:
             continue
-        if cmd['flags'] is not None and ((cmd['flags'] & (FLAG_OBSOLETE | FLAG_DEPRECATED)) != 0):
+        flags = cmd.get('flags')
+        if (flags is not None and
```

Comment:
I mean, it is not necessary to use `&`, `|`, or `<<` here. Regular Python operators would do just fine.